### ISSUE TITLE
Importing non-vendor assets?

### ISSUE THREAD

**Issue by oskarrough** [ember-cli/ember-cli#757]
Hi, how would you include non-vendor assets? For instance, a custom modernizr.js.

Does the server map the /public somehow? In Brocfile.js import only worked for the vendor dir.

**Comment by benoittgt** [comment]
Sadly [the link for assets in test](https://github.com/ember-cli/ember-cli/issues/757#issuecomment-49659860) is dead. 

[I'm trying to import i18n-js](https://github.com/fnando/i18n-js/issues/366). Probably the wrong way, but can't found any resources on tests assets

**Comment by cyril-sf** [comment]
>  are you saying you'd like it to be prepended to my-appname.js?

That's actually what I'm trying to achieve. The reason: I'm migrating a global based app written in CoffeeScript to Ember-CLI. So right now, pretty much all the app is built w/ custom code in `ember-cli-build.js`, to get something like `my-old-app.js`. In Ember-CLI land, I only ported the router, and I wrote a custom resolver to deal w/ both parts. The app works well.

That approach forces me to load all the old code between `vendor.js` and `my-appname.js`. While I can easily update the `app/index.html` file to reflect that, it doesn't work when running tests.

Am I taking a wrong approach?

cc @stefanpenner @rwjblue
